_layouts/main.html

```diff
@@ -62,22 +62,15 @@ <h2>Meetings</h2>
 
 
   <div class="col-lg-4">
-    <h2>Analysis Ecosystems Workshop II</h2>
+    <h2>PyHEP 2022</h2>
     <p>
-      Five years after the original Analysis Ecosystems Workshop in 
-      Amsterdam, the HSF and IRIS-HEP are organising the second
-      <a href="https://indico.cern.ch/e/aew2">Analysis Ecosystems Workshop</a>
-      at IJCLab in Paris from 23-25 May.
+      Registration for the <a href="https://indico.cern.ch/e/pyhep2022">PyHEP 2022 workshop</a> is now <a href="https://indico.cern.ch/event/1150631/registrations/82480/">open</a>.
     </p>
-    <p>
-      Much has happened since 2017 and
-      now is an ideal time to review progress and look at the future of
-      analysis at HL-LHC, DUNE, EIC, FAIR and other HEP experiments
-      at future facilities.
-    </p>
-    <p>
-      Registration is <a href="https://indico.cern.ch/event/1125222/registrations/">open now</a>
-      - sign up now to be part of the discussion on our analysis future.
+    <img src="/images/pyhep/PyHEP_logo_template_white.png"
+    alt="PyHEP-logo" style="display: block; width:55%; margin-left: auto; margin-right: auto;">
+    <p> After the huge success of PyHEP as virtual events since 2020, the workshop is back, again as a virtual event. The call for abstracts is
```

Comment:
Can I suggest a very small rephrasing to avoid saying virtual event twice:

> After the huge success of virtual PyHEP workshops since 2020, we are back, again as a virtual event

Reply:
Better indeed. Just committed the change. Thanks!